src/Raven.Server/Documents/TimeSeries/TimeSeriesStats.cs

```diff
@@ -326,7 +326,7 @@ public string GetTimeSeriesNameOriginalCasing(DocumentsOperationContext context,
         public (long Count, DateTime Start, DateTime End) GetStats(ref TableValueReader tvr)
         {
             var count = DocumentsStorage.TableValueToLong((int)StatsColumns.Count, ref tvr);
-            var start = new DateTime(Bits.SwapBytes(DocumentsStorage.TableValueToLong((int)StatsColumns.Start, ref tvr)));
+            var start = new DateTime(Bits.SwapBytes(DocumentsStorage.TableValueToLong((int)StatsColumns.Start, ref tvr)), DateTimeKind.Utc);
             var end = DocumentsStorage.TableValueToDateTime((int)StatsColumns.End, ref tvr);
```

Comment:
why is end created in a different way than start?

Shouldn't this be:
```
var start = DocumentsStorage.TableValueToDateTime((int)StatsColumns.Start, ref tvr);
```

Reply:
what this fixed?